Source/WebKit/NetworkProcess/mac/com.apple.WebKit.NetworkProcess.sb.in

```diff
@@ -742,3 +742,7 @@
         (allow syscall-mach
             (machtrap-number MSC_mach_msg2_trap))))
 #endif // HAVE(SANDBOX_MESSAGE_FILTERING)
+
+#if USE(APPLE_INTERNAL_SDK)
```

Comment:
Does this need a `__has_include`? Same question for the other `sb.in`.

Reply:
Added — thanks for the review!